regression/strings/smoke_tests/performance.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python
+import time
+from subprocess import check_output
+
+git_output=check_output(["git", "show", "HEAD"])
+commit=git_output.split('\n')[0]
+
+commit_id=time.strftime("%Y_%m_%d__%H_%M_%S")+"__"+commit[7:]
+
+process = check_output(["grep", "^Runtime\ decision\ procedure", "-R"])
```

Comment:
Might be wrong but I don't think those escaped spaces are doing anything (as in, Python will already submit "Runtime decision procedure" as a single argument, and grep will regard "\ " and " " as the same AFAIK)